HowToREST.md

```diff
@@ -0,0 +1,104 @@
+# How to Call Google APIs: REST Edition
+
+_For many developers, the easiest way to call a Google API is with one of our client libraries. But occasionally someone may prefer to make API calls directly - perhaps from a language or environment that we don’t support or using a different networking library or tool. Here we’ll show you how to do it._
+
+This page focuses on calling Google APIs using JSON REST interfaces. Most of these APIs are also available as Protocol Buffer-based 
+RPC services. For more on that, see [How to Call Google APIs, RPC Edition](/HowToRPC).
+
+## What you’ll need
+
+### An API definition
+Most Google APIs are available as JSON REST services. These APIs are formally described by the [Google API Discovery Service](https://developers.google.com/discovery/) in a JSON representation known as the [Discovery Document](https://developers.google.com/discovery/v1/reference/apis) format.
+
+For an example, see the [Cloud Natural Language API](https://cloud.google.com/natural-language/), which is defined by [this Discovery Document](https://language.googleapis.com/$discovery/rest?version=v1). Human-readable documentation is in the [Google Cloud Natural Language API reference](https://cloud.google.com/natural-language/docs/reference/rest/). We’ll call the [AnalyzeEntities](https://cloud.google.com/natural-language/docs/reference/rest/v1/documents/analyzeEntities) service, which takes a block of text as input and returns a list of names and nouns that it finds in the text along with some interesting properties of each entity.
+
+### JSON
+Google REST APIs use JSON for most message responses and POST bodies. Quoting [www.json.org](https://www.json.org/):
+
+> JSON (JavaScript Object Notation) is a lightweight data-interchange format. It is easy for humans to read and write. It is easy for machines to parse and generate. It is based on a subset of the [JavaScript Programming Language](http://crockford.com/javascript/), [Standard ECMA-262 3rd Edition - December 1999](http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-262.pdf). JSON is a text format that is completely language independent but uses conventions that are familiar to programmers of the C-family of languages, including C, C++, C#, Java, JavaScript, Perl, Python, and many others. These properties make JSON an ideal data-interchange language.
+
+### A way to make API requests
+Google REST APIs are designed to be called by any standards-compliant HTTP client, including the [curl](https://curl.haxx.se/) command-line tool and library.
+
+### Authentication
+To use Google APIs, a client needs to use API keys for unauthenticated requests or OAuth tokens for authenticated requests. For more information, see [the Google Cloud Authentication Overview](https://cloud.google.com/docs/authentication/).
```

Comment:
"use API keys for unauthenticated requests" is confusing. Fairly certain it is technically correct, but from a customer's perspective, they are authenticating _as themselves_ with their API key.

Reply:
Simplifying to "To use Google APIs, a client needs to authenticate with an API key or an OAuth token."